lms/routes.py

```diff
@@ -64,6 +64,8 @@ def includeme(config):
     )
     config.add_route("canvas_api.sync", "/api/canvas/sync", request_method="POST")
 
+    config.add_route("api.grant_token", "/api/grant_token", request_method="GET")
```

Comment:
Does anyone have recommendations regarding the API name and path or is this good?

Reply:
I think this seems correct as the resource you are trying to retrieve is a grant token and it's a GET request here...seems about right to me.

Reply:
Yeah, this seems fine. It's a bit ugly perhaps (would `grant-token` be nicer than `grant_token` in a URL?). And should it be under `/api/auth/`? Is "grant token" the best name? That does't really get across what this is: a token for the client to use to log in to h.

But ultimately this is a private API only used by LMS's own frontend code, so URL design probably doesn't matter that much, and is easy to change